src/app/gen-queries.ts

```diff
@@ -82,7 +82,7 @@ export const SampleQueries: SampleQuery[] = [
     "category": "Users",
     "method": "GET",
     "humanName": "user by email",
-    "requestUrl": "https://graph.microsoft.com/v1.0/users('{user-mail}')",
+    "requestUrl": "https://graph.microsoft.com/v1.0/users/{coworker-mail}",
```

Comment:
I like the original user-mail token since you can put in your own upn. Colloquially speaking, in large orgs, unknown people in the org might not be considered co-worker. User is consistent with the Graph paradigm. I suggest you change it back.